### PR TITLE
c: small fixes/additions

### PR DESCRIPTION
**Pull request by mnunberg**

### CODE REVIEW

**Comment by jamiltz** [comment]
@mnunberg I'm the gatekeeper on this repo while Amarantha is away :)

I'll read the changes and check for typos..but can't comment on the technical side of things.
Let me know when you would like me to pull and build.

**Comment by mnunberg** [comment]
James, the build should be fine. You can pull this at your earliest convenience, though it is probably not a matter of urgency.

File: content/sdk/c/handling-error-conditions.dita [hunk 1]
```diff
@@ -55,6 +55,50 @@
...
       <section conref="../shared/couchbase-errors.dita#toplevel/begin" conrefend="../shared/couchbase-errors.dita#toplevel/end"/>
+        <section>
+            <title>Inspecting HTTP (N1QL, MapReduce, FTS) Errors</title>
+            <p>Many of the SDK's APIs (<apiname>lcb_n1ql_query</apiname>,
+                    <apiname>lcb_view_query</apiname>, <apiname>lcb_fts_query</apiname>) use HTTP
+                internally. Unlike Key-Value based APIs which use a binary protocol and have a fixed
+                internal format for errors, APIs using JSON-over-HTTP can report multiple errors
+                anywhere during execution. The C SDK itself can only report a single top-level error
+                code (i.e. the <parmname>rc</parmname> within the response structure).</p>
+            <p>In cases where an HTTP API failed with a non-successful (<i>2xx</i>) HTTP reply, the
```
Comment (jamiltz) [inline content/sdk/c/handling-error-conditions.dita]: Did you mean to write "other than 2xx"?

File: content/sdk/c/view-queries-with-sdk.dita [hunk 2]
```diff
@@ -16,28 +16,24 @@
...
-            </codeblock>This
-                callback is invoked for each row. The end of the query (when no more rows to return) is
-                signaled by having the <codeph>LCB_RESP_F_FINAL</codeph> bit set in the response's
-                <codeph>rflags</codeph> field.</p>
+    if (rv->docresp) {
+        printf("   Document for response. RC=0x%X. CAS=0x%llx\n",
+            rv->docresp->rc, rv->docresp->cas);
+    }
+}</codeblock>This
+                callback is invoked for each row. The end of the query (when no more rows to return)
```
Comment (jamiltz) [inline content/sdk/c/view-queries-with-sdk.dita]: should it read "when **there are** no more rows to return"?

**Comment by mnunberg** [comment]
Fixed!

**Comment by ingenthr** [comment]
rebased, tested, merged

**Pull request closed without merging**